WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1741,9 +1741,7 @@ public void onPositiveClicked(@NonNull String instanceTag) {
                 mZendeskHelper.createNewTicket(this, Origin.DISCARD_CHANGES, mSite);
                 break;
             case TAG_PUBLISH_CONFIRMATION_DIALOG:
-                mPost.setStatus(PostStatus.PUBLISHED.toString());
-                mPostEditorAnalyticsSession.setOutcome(Outcome.PUBLISH);
```

Comment:
Setting the status of the Post immediately made the `isFirstTimePublish()` check to fail so, we moved this code  here https://github.com/wordpress-mobile/WordPress-Android/pull/9544/files#diff-38b6aea9cfff5f3e0e4b7389b44de602R2020